services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from django.utils.dateparse import parse_date
+from db.models import Movie, CinemaHall, MovieSession
+
+
+def create_movie_session(movie_show_time: int |float,
+                         movie_id: id,
+                         cinema_hall_id: int) -> Movie | CinemaHall:
+    movie=Movie.objects.get(id=movie_id)
+    cinema_hall=CinemaHall.objects.get(id=cinema_hall_id)
+    return Movie.objects.create(
+        movie_show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id)
```

Comment:
The `create` method is being called on `Movie.objects`, which is incorrect. It should be `MovieSession.objects.create()` to create a `MovieSession` object.